charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -146,6 +153,78 @@ spec:
         securityContext:
 {{ toYaml .Values.containerSecurityContext | indent 10 }}
 {{- end }}
+        {{-  if .Values.kubeRBACProxy.enabled  }}
+      - name: kube-rbac-proxy-http
+        args:
+        {{-  if .Values.kubeRBACProxy.extraArgs  }}
+        {{- .Values.kubeRBACProxy.extraArgs | toYaml | nindent 8 }}
+        {{-  end  }}
+        - "--secure-listen-address=:{{ .Values.service.port | default 8080}}"
```

Comment:
Is there a reason to quote this line and not the others?

Reply:
I added this because my IDE was linting it, but it looks like `helm` and `ct lint` are fine with it, I'll remove the quotes, must be an IDE bug
<img width="463" alt="image" src="https://user-images.githubusercontent.com/1838835/210972156-98b43cff-9854-489a-b1af-3a4209e5b8f8.png">